Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqTable.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqTable" {
```

Comment:
I was planning to add that in as I filled in the other cmdlets.  At least, that's what I did with the Datasets.  Are you saying using the bq command or something external to generate/destroy them?